src/app/components/HoldConfirmation/HoldConfirmation.jsx

```diff
@@ -115,66 +116,77 @@ class HoldConfirmation extends React.Component {
     }
 
     return (
-      <div id="mainContent">
-        <div className="page-header">
-          <div className="content-wrapper">
-            <Breadcrumbs
-              query={this.props.searchKeywords}
-              type="holdConfirmation"
-              title={title}
-              url={id}
-            />
+      <main id="mainContent" className="main-page">
```

Comment:
So I updated the header again along with the whole page's <main>. And the result is shown as the screenshot below.

![screencapture-local-nypl-org-3001-research-collections-discovery-beta-hold-confirmation-b14870253-i12348551-1501176284274](https://user-images.githubusercontent.com/3506922/28683603-29213316-72cf-11e7-9d26-592d91020692.png)

I followed the example from toolkit, here, to make the DOM
https://nypl.github.io/design-toolkit/request-confirmation.html

And I followed the google doc from Courtney, to make the headings
https://docs.google.com/document/d/1DPkA2MDQrhFEis0LeK7Q3MeCYOVr1ZaOCyZmecG1oYU/edit

except the heading of "Research Discovery (beta)" as it is not indicated in the google doc.

One issues I have noticed is that "Research Discovery (beta)"  and the content below are not aligned on the left.
Also, I want to confirm again if the visual are the one we want. Thanks!!!  @ricardoom @courtneymcgee